assets/src/js/modules/text-decoration.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Text Decoration
+ * For any elements where we used the editor to apply a custom link color,
+ * update the individual links 
+ *  
+ */
+
+const PROPERTY_NAME = '--color-text-decoration-rgb';
+
+/**
+ * Main - function which changes the underline color for a link based on the text color
+ *
+ * @return {void}
+ */
+export default function main() {
+	changeTextDecorationColor();
+	
+	if ( 0 < document.querySelectorAll( '.wp-block-cata-toc' ).length ) {
```

Comment:
@jessica-townsend I SWEAR I sent feedback on this yesterday. Sorry if you've been waiting on me.

The previous comment said, I think the link styles script should know as little as possible about the Table of Contents.

So while it makes total sense to omit the event if there's no TOC, it creates maintanence since we have to keep this script up to date with the selector for the TOC.

I think the event listener should just always be added.

Reply:
No worries! I'll remove that check